src/main/java/de/szut/lf8_project/application/ProjectApplicationService.java

```diff
@@ -148,6 +151,18 @@ public List<ProjectView> getAllProjects() {
         }
     }
 
+    private ProjectRole getEmployeeRoleInProject(EmployeeId employeeId, Project project) {
+        return project.getTeamMembers().stream().filter(member -> member.getEmployeeId().equals(employeeId)).findFirst().get().getProjectRole();
```

Comment:
ja, dieser .get() call **müsste** implizit immer klappen. just in case und um im Parteiprogramm "gib `null` keine Chance" zu bleiben würde ich dafür plädieren `.isPresent()`zu checken und im fehlerfall einen 500er zu werfen.

edit: bzw getOrthrow

Reply:
Jo stimmt, könnte ja sonst theoretisch auch noch wer anders mal aufrufen.
War vorher auch nicht gelb, da es keine externe private Methode war.
Ist gefixt